queue/queue.css

```diff
@@ -7,7 +7,7 @@
 
 .queue-count {
 	font-size: 2rem;
-	font-family: 'Tungsten A','Tungsten B','Arial Narrow',Arial,sans-serif;
+	/* font-family: 'Tungsten A','Tungsten B','Arial Narrow',Arial,sans-serif; */
```

Comment:
Should we just remove these commented-out lines?

Reply:
Yes.